### PR TITLE
fix(contextadd): add ability to enable features on page

### PR DESCRIPTION
**Pull request by dgutride**

### CODE REVIEW

**Comment by fabric8cd** [comment]
@dgutride fabric8/fabric8-ui:SNAPSHOT-PR-1723-1 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1723-fabric8-ui.badger.fabric8.io

File: src/app/exp-feature-page/exp-feature-page.component.ts [hunk 1]
```diff
@@ -1,19 +1,61 @@
...
+      profile.contextInformation.experimentalFeatures = {};
+    }
+    profile.contextInformation.experimentalFeatures["enabled"] = true;
+
+    this.gettingStartedService.update(profile).subscribe(() => {
+      // reset boolean in case they come back in here later
+      this.enableFeatures = false;
+      this.onExperimentalFeaturesEnabled.emit(true);
+      this.notifications.message({
+        message: `Profile updated!`,
```
Comment (mindreeper2420) [inline src/app/exp-feature-page/exp-feature-page.component.ts]: almost wondering if this should be 'Experimental Feature Enabled', or something along those lines?

Comment (dgutride) [inline src/app/exp-feature-page/exp-feature-page.component.ts]: good idea, I'll update

**Comment by fabric8cd** [comment]
@dgutride fabric8/fabric8-ui:SNAPSHOT-PR-1723-2 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1723-fabric8-ui.badger.fabric8.io

**Comment by fabric8cd** [comment]
@dgutride fabric8/fabric8-ui:SNAPSHOT-PR-1723-3 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1723-fabric8-ui.badger.fabric8.io

**Pull request merged**